frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -131,13 +131,36 @@ export const startPipeline: KebabAction = (
   },
 });
 
-export const rerunPipeline: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => {
-  // The returned function will be called using the 'kind' and 'obj' in Kebab Actions
+type RerunPipelineData = {
+  pipelineRunRedirect?: boolean;
+};
+export const rerunPipeline: KebabAction = (
+  kind: K8sKind,
+  pipelineRun: PipelineRun,
+  resources: any,
```

Comment:
resources is not being used in the method

Reply:
Nope, but it's the signature of the `KebabAction`, and I need the `customData` for my callbacks:
```
export type KebabAction = (
  kind: K8sKind,
  obj: K8sResourceKind,
  resources?: any,
  customData?: any,
) => KebabOption;
```